### PR TITLE
Initial draft of working agreement

### LINKED ISSUE #265: Draft Working Agreement
It's important that we have a team working agreement to set ground rules for how we will work together as a team.

- [x] Initial Draft of WA
- [x] Second Draft of WA
- [ ] Team Review
- [ ] Final Draft of WA (final for purposes of this issue)
- [ ] Post WA to repo and link to it in contributing.md (@ryancrawcour does this seem right to you?)

### PR DESCRIPTION
**Pull request by brandynbrown**
## Type of PR

- [x] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Review notes
Please review and make corrections/commits to this branch
pay extra attention to the "how we code together" section

## Issues Closed or Referenced

- Closes #265

### CODE REVIEW

**Comment by brandynbrown** [comment]
Can whoever makes the next updates please address my comments? (That person may be me lol)

File: .github/working_agreement.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+**N**egotiable - Is the scope negotiated to enable completion?  
+**V**aluable - Is the value to the user or customer clear?  
+**E**stimatable - Can the work be estimated? Are there unknowns, dependencies, barriers? Do we lack domain or technical knowledge?  
+**S**ize appropriately - Can it be completed in the iteration?  
+**T**estable - What are the tests to know the work is done?
+
+##### User Story Issues to Avoid
+
+- Describing a task
+- UI too soon
```
Comment (sivamu) [inline .github/working_agreement.md:63]: What does this mean? Is it actually the User Interface being made too soon?

File: .github/working_agreement.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+- Write in an inactive voice
+- Splitting too often in the iteration
+- Thinking too far ahead
+- Interdependent Stories
+- Too many details
+- Goldplating
```
Comment (sivamu) [inline .github/working_agreement.md:69]: I had to look this up, not sure how many folks have encountered this term. A link to the wikipedia article would be helpful - https://en.wikipedia.org/wiki/Gold_plating_(project_management)

File: .github/working_agreement.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+### Branch Strategy
+
+- `main` branch will be used as the production branch
+  - It will be locked, requiring a PR to make commits.
+  - It will be shippable at any time.
+  - Releases will occur through a merge of `dev` into `main`
+- Branches will be used for story or bug-fix work.
+  - Naming convention will include alias with a short description of the story or bug (e.g., `dsturgell-create-working-agreement` or `jofultz-deploy-script-error-on-box`)
+  - Commits should always have a short but descriptive message explaining what is changing.
```
Comment (kev-ms) [inline .github/working_agreement.md:131]: Probably worth mention that `dev` is a protected working branch for CI. and `main` should always have ready to release code.

**Pull request merged**